algorithms/estonian.sbl

```diff
@@ -0,0 +1,256 @@
+/* Estonian stemmer version 1.3
+
+Made by Linda Freienthal in January 2019.
+
+*/
+
+routines (
+	mark_regions
+	LONGV
+	special_noun_endings
+	case_ending
+	emphasis
+	plural_three_first_cases
+	remove_double_kpt
+	double
+	undouble
+	i_plural
+	degrees
+	substantive
+	verb_exceptions
+	verb
+	nu
+)
+
+stringdef a" hex 'E4' //a-umlaut ä
+stringdef o" hex 'F6' //o-umlaut ö
+stringdef o' hex 'F5' //o with tilde õ
+stringdef u" hex 'FC' //u-umlaut ü
```

Comment:
You should use the newer notation here - that may fix at least some of the test failures.  E.g.:

```
stringdef a"   '{U+00E4}'
```

Reply:
Fixed by 2ff0d924943f484995c74157672ab413a1aa5774.